src/frontend/src/statement.rs

```diff
@@ -70,7 +70,7 @@ impl StatementExecutor {
         }
     }
 
-    async fn execute_sql(&self, stmt: Statement, query_ctx: QueryContextRef) -> Result<Output> {
+    pub async fn execute_sql(&self, stmt: Statement, query_ctx: QueryContextRef) -> Result<Output> {
```

Comment:
Why `pub` this method? The entry point for executing SQL statements is expected to be `execute_stmt`.

Reply:
we need to use this for an extension api.